File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/RetailIdAllotment.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class RetailIdAllotment {
    @JsonProperty("Allotment")
    public Integer allotment;
    @JsonProperty("FacilityId")
    public Integer facilityId;
    @JsonProperty("FacilityLicenseNumber")
    public String facilityLicenseNumber;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/RetailIdAllotment.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class RetailIdAllotment(
    @JsonProperty("Allotment")
    val allotment: Int? = null,
    @JsonProperty("FacilityId")
    val facilityId: Int? = null,
    @JsonProperty("FacilityLicenseNumber")
    val facilityLicenseNumber: String? = null
)
